.github/workflows/linters.yaml

```diff
@@ -81,3 +81,28 @@ jobs:
           github_token: ${{ secrets.github_token }}
           reporter: github-pr-review # Change reporter
           android: true
+  qmllint:
+    name: Run QML-Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Install Qt
+        shell: bash
+        run: |
+          python3 -m pip install aqtinstall
```

Comment:
Now that `ubuntu-latest` has migrated to 22.04, I think you can install `qmtlint` using `apt get install qt6-declarative-dev-tools` which is probably a bit faster and more reliable than `aqt`


[qt6-declarative-dev-tools](https://packages.ubuntu.com/kinetic/qt6-declarative-dev-tools)`